source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -15,13 +15,43 @@ Overview
 
 .. _sink-configuration-write-model-strategy-description-start:
 
-Use the strategies in the following table to specify how the {+sink-connector+}
-writes data into MongoDB. You can specify a write strategy with the following
+You can set configuration properties to specify how the
+{+sink-connector+} writes data into MongoDB. The following sections
+describe the configuration properties that you can set to customize this
+behavior.
+
+Write Model
+~~~~~~~~~~~
+
+Set the ``writemodel.strategy`` configuration property to specify how
+the sink connector writes data when it receives a new sink record.
```

Comment:
Suggestion:

I think "new" is an unnecessary adjective for descriptions of the sink record and tombstone event, and would omit it.